src/distrs/wilcox.jl

```diff
@@ -0,0 +1,155 @@
+#=
+The wilcox distribution is equivalent to the problem
+of finding the number of subsets with size nx
+of {1,2,...,nx,nx+1,...,nx+ny} summing to (U-sum(1:nx))
+relative to the total number of subsets with size nx.
+The empty subset is defined to sum to zero.
+This can be calculated using the recursion:
+either nx+ny is in the subset in witch case we need to calculate
```

Comment:
```suggestion
either nx+ny is in the subset in which case we need to calculate
```

Reply:
Unrelated, but do you find the comment clear about what is being calculated?